### ISSUE TITLE
config.mk MKLDNN=0 not working in Makefile

### ISSUE THREAD

**Issue by lanking520** [apache/incubator-mxnet#13779]
I tried to use `config.mk` to set MKLDNN=0 and it seemed not to work
### step to reproduce:
```
echo "USE_MKLDNN=0" >> config.mk
make -j4
```
Change on `Makefile`
```
# use customized config file
include $(config)
$(info $$USE_MKLDNN is [${USE_MKLDNN}])
ifndef $(USE_MKLDNN)
$(info MKLDNN is not defined)
endif
```
Result
```
$USE_MKLDNN is [0]
MKLDNN is not defined
```
Will raise a PR to fix this

**Comment by lanking520** [comment]
@zachgk @szha @azai91 @mseth10

**Comment by szha** [comment]
Fixed in #13775
